website/docs/cloud-docs/run/ui.mdx

```diff
@@ -65,7 +65,7 @@ To help you review proposed changes, Terraform Cloud can automatically run [spec
 
 ### Viewing Pull Request Plans
 
-Speculative plans don't appear in a workspace's list of normal runs. Instead, Terraform Cloud adds a link to the run in the pull request itself, along with an indicator of the run's status.
+Speculative plans can be viewed in a workspace's list of normal runs. Additionally, Terraform Cloud adds a link to the run in the pull request itself, along with an indicator of the run's status.
 
 A single pull request can include links to multiple plans, depending on how many workspaces are connected to the destination branch. If a pull request is updated, Terraform Cloud will perform new speculative plans and update the links.
```

Comment:
```suggestion
A single pull request can include links to multiple plans, depending on how many workspaces connect to the destination branch. If you update a pull request, Terraform Cloud performs new speculative plans and update the links.
```

Suggest changing "are connected" to "connect" and "is updated" to "you update" to avoid a passive voice style issue. Also suggest changing "will perform" to "performs" to avoid future tense usage.